examples/dataservice-write/CMakeLists.txt

```diff
@@ -60,12 +60,94 @@ if (IOS)
         ${CF_NETWORK_FRAMEWORK}
         ${UIKIT_FRAMEWORK}
     )
+elseif(ANDROID)
```

Comment:
Please make sure that all local variables have the `EDGE_SDK_` prefix so that we do not create conflicts with other projects including us.

Reply:
Done for local variables. For the ANDROID -  actually, it is default variable  which is set by toolchain file: https://cmake.org/cmake/help/v3.7/variable/ANDROID.html

Reply:
Only EDGE SDKs own cmake variables of course. Everything injected from outside needs to stay the same.